send-transaction-service/src/send_transaction_service.rs

```diff
@@ -305,9 +352,67 @@ impl SendTransactionService {
             }
         });
 
+        if batched_transactions.is_empty() {
```

Comment:
prefer avoiding the early return here.  invert the logic and wrap the batch handling instead

Reply:
Done